pypeit/scripts/coadd_2dspec.py

```diff
@@ -152,14 +152,31 @@ def main(args):
         all_spec2d['meta']['bkg_redux'] = bkg_redux
         all_spec2d['meta']['find_negative'] = find_negative
 
+        # get only_slits and exclude_slits if they are set
```

Comment:
Why not just put these if/then statements and this control here internal to the parser, and if only_slits is set it returns the values you want and otherwise it returns None. That would clean up the control flow here

Reply:
"internal to the parser" you mean in `pypeitpar.py`? I can do that to make `par['coadd2d']['only_slits']` and `par['coadd2d']['exclude_slits']` mutually exclusive, but `parse_slitspatnum` still needs to be here.